<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>查找目标为：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>替换为：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="searchText"/>
          </item>
          <item>
           <widget class="QLineEdit" name="targetText"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="chCaseSensitive">
          <property name="text">
           <string>区分大小写</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>方向</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="rbUp">
               <property name="text">
                <string>向上（U)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbDown">
               <property name="text">
                <string>向下(D)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btFindNext">
        <property name="text">
         <string>查找下一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btReplaceAll">
        <property name="text">
         <string>全部替换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btReplace">
        <property name="text">
         <string>替换</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
